### PR TITLE
PostgreSQL Flexible Server Preview Swagger API changes

### PR DESCRIPTION
**Pull request by pradeepkintali**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by rajsell** [comment]
@pradeepkintali can you incorporate the changes for the readme file from this 
PR https://github.com/Azure/azure-rest-api-specs/pull/10567

**Comment by pradeepkintali** [comment]
> @pradeepkintali can you incorporate the changes for the readme file from this
> PR #10567
Updated the readme files.

**Review by pilor — APPROVED** [review]
Just moving api-versions.  Migrated APIs were not changed

**Comment by QingqingYuan2** [comment]
@NullMDR ARM has singed off this PR. Can we merge this as soon as possible? We have other products blocking by this check in.

File: specification/mysql/resource-manager/readme.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Management.MySQL
-  output-folder: $(csharp-sdks-folder)/postgresql/Microsoft.Azure.Management.PostgreSQL/src/postgresql/Generated
+  output-folder: $(csharp-sdks-folder)/postgresql/Microsoft.Azure.Management.MySQL/src/postgresql/Generated
```
Comment (gechris) [inline specification/mysql/resource-manager/readme.md:164]: Path should be 
output-folder: $(csharp-sdks-folder)/**mysql**/Microsoft.Azure.Management.MySQL/src/postgresql/Generated

Comment (pradeepkintali) [reply specification/mysql/resource-manager/readme.md]: updated the path folder.

**Comment by PhoenixHe-msft** [comment]
Please go and fix the lint error in https://github.com/Azure/azure-rest-api-specs/pull/10770#issuecomment-691390351

**Comment by pradeepkintali** [comment]
> Please go and fix the lint error in [#10770 (comment)](https://github.com/Azure/azure-rest-api-specs/pull/10770#issuecomment-691390351)

Done all are fixed now.

**Comment by QingqingYuan2** [comment]
@PhoenixHe-msft All check has been passing now. Can we get this merged ASAP?

**Pull request merged**